source/docs/software/vision-processing/wpilibpi/walkthrough-video.rst

```diff
@@ -1,7 +1,7 @@
 .. include:: <isonum.txt>
 
-A Video Walkthrough of using FRC Vision with the Raspberry Pi
-=============================================================
+A Video Walkthrough of using WPILibPi with the Raspberry Pi
```

Comment:
I'm not sure about changing this one, since the presentation was before the rename and still uses FRC Vision. Perhaps a note about the name change.

Reply:
added note